bigtable-hbase-1.x-parent/bigtable-hbase-tools/README.md

```diff
@@ -0,0 +1,104 @@
+#HBase Tools
+
+This folder contains tools to help HBase users with migrations to Cloud Bigtable.
+Pipelines to import and export data are under [bigtable-beam-import](bigtable-dataflow-parent/bigtable-beam-import/README.md).
+
+## Setup 
+
+To use the tools in this folder, you can download them from the maven repository, or
+you can build them using Maven.
+
+[//]: # ({x-version-update-start:bigtable-dataflow-parent:released})
+### Download the jars
+
+todo: get correct link, also should I provide a wget command?
+
+Download [the Bigtable tools jars](http://search.maven.org/remotecontent?filepath=com/google/cloud/bigtable/bigtable-beam-import/1.14.1/bigtable-beam-import-1.14.1-shaded.jar), which is an aggregation of all required jars.
+
+### Build the jars
+Go to the top level directory and build the repo then return to this sub directory.
```

Comment:
I think we should remove this from docs and as a requirement. The dependencies should come from maven repo and not from building locally.

Reply:
I can make it clearer that it's just an alternative (for those who are contributing or those who want to make any changes.) Let me know your thoughts